Assets/Scripts/CharacterController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using UnityEngine;
+
+public class CharacterController : MonoBehaviour
```

Comment:
Confused about what this class is for. It looks like Player.cs is your move controller. Maybe this is leftover and needs to be deleted?